components/Match/Match.jsx

```diff
@@ -79,27 +85,28 @@ class RequestLayer extends React.Component {
         <MatchTable columns={benchmarksColumns(this.props.match)} />
         <MatchTable columns={overallColumns} />
         <MatchTable columns={laningColumns} />
-        {this.props.match.players.length ? this.props.match.players.map(p =>
-          (<table>
-            <thead>
-              <tr>
-                <th>Ability</th>
-                <th>Casts</th>
-                <th>Hits</th>
-                <th>Damage</th>
-              </tr>
-            </thead>
-            <tbody>
-              {p.ability_uses && p.hero_hits && p.damage_inflictor ? Object.keys(p.ability_uses).map(k =>
-                (<tr>
-                  <td>{k}</td>
-                  <td>{p.ability_uses[k]}</td>
-                  <td>{p.hero_hits[k]}</td>
-                  <td>{p.damage_inflictor[k]}</td>
-                </tr>)) : <tr />}
-            </tbody>
-          </table>)) : <div />
+        <MatchTable columns={purchaseColumns} />
+        <Tabs>
+        {this.props.match.players.map(p =>
+          (
+          <Tab icon={<img src={`${API_HOST}${constants.heroes[p.hero_id].img}`} width={30} role="presentation" />}>
+            <Table
+              data={Object.keys(p.ability_uses || {}).map(k => ({
+                name: k,
+                casts: (p.ability_uses || {})[k],
+                hero_hits: (p.hero_hits || {})[k],
+                damage_inflictor: (p.damage_inflictor || {})[k],
+              }))}
+              columns={[{ displayName: 'Ability', field: 'name' },
+            { displayName: 'Casts', field: 'casts', sortFn: true },
+            { displayName: 'Hits', field: 'hero_hits', sortFn: true },
+            { displayName: 'Damage', field: 'damage_inflictor', sortFn: true }]}
+            />
+          </Tab>
+          ))
         }
```

Comment:
It would probably be worthwhile to make this a component or higher order component (similar to withPagination and PaginatedTable). I could see using this in other areas at least.